File: python/udemy/3-estrutura-sequencial/retangulo.py
# Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.

import math

base = float(input('Base do retângulo: '))
altura = float(input('Altura do retângulo: '))

area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(base**2 + altura**2)

print(f'Área: {area}, perímetro: {perimetro}, diagonal: {round(diagonal, 2)}')

File: c/udemy/3-sequencial/retangulo.c
// Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.

#include <stdio.h>
#include <math.h>

int main() {

    double base, altura, area, peri, diag;

    printf("Base: ");
    scanf("%lf", &base);

    printf("Altura: ");
    scanf("%lf", &altura);

    area = base * altura;
    peri = 2 * base + 2 * altura;

    diag = sqrt(pow(base, 2) + pow(altura, 2));

    printf("\nÁrea = %.4lf", area);
    printf("\nPerímetro = %.4lf", peri);
    printf("\nDiag = %.4lf", diag);

    return 0;
}